focal/4_gs1_identification.html

```diff
@@ -0,0 +1,336 @@
+            </li>
+            <li>
+                  A trade item manufactured and sold by Healthy Tots, represented as a <a
+                        href="https://www.gs1.org/standards/gs1-digital-link" rel="nofollow">GS1 Digital Link</a> URI
+            </li>
+      </ul>
+      <h5>Issuer</h5>
+      <ul>
+            <li>
+                  For the GS1 Prefix license Verifiable Credential, the issuer is GS1 Global Office.
+            </li>
+            <li>
+                  For the GS1 Company Prefix license Verifiable Credential, the issuer is GS1 Utopia, which is the subject of
+                  the corresponding GS1 Prefix license Verifiable Credential.
+            </li>
+            <li>
+                  For the trade item Verifiable Credential, the issuer is Healthy Tots, which is the subject of the
+                  corresponding GS1 Company Prefix license Verifiable Credential.
+            </li>
+      </ul>
+      <h5>Subject</h5>
+      <ul>
+            <li>
+                  For the GS1 Prefix license Verifiable Credential, the subject is GS1 Utopia.
+            </li>
+            <li>
+                  For the GS1 Company Prefix license Verifiable Credential, the subject is Healthy Tots.
+            </li>
+            <li>
+                  For the trade item Verifiable Credential, the subject is the GTIN represented as a <a
+                        href="https://www.gs1.org/standards/gs1-digital-link" rel="nofollow">GS1 Digital Link</a> URI.
+            </li>
+      </ul>
+      <h5>Holder</h5>
+      <ul>
+            <li>
+                  For the GS1 Prefix license Verifiable Credential, the holder is GS1 Utopia.
+            </li>
+            <li>
+                  For the GS1 Company Prefix license Verifiable Credential, the holder is Healthy Tots.
+            </li>
+            <li>
+                  For the trade item Verifiable Credential, the holder is Healthy Tots.
+            </li>
+      </ul>
+      <h5>Verifier</h5>
+      <ul>
+            <li>
+                  Sell Anything &amp; Everything, a trading partner of Healthy Tots that needs to validate the identification
+                  of an object (typically a trade item) and the data associated with it.
+            </li>
+      </ul>
+      <h4>Validation Requirements</h4>
+      <p>
+            A GS1 Prefix license Verifiable Credential is valid if it is issued by GS1 Global Office.
+      </p>
+      <p>
+            A GS1 Company Prefix license Verifiable Credential is valid if:
+      </p>
+      <ul>
+            <li>
+                  the issuer is the same as the subject of the "extendsCredential";
+            </li>
+            <li>
+                  the GS1 Company Prefix in "licenseValue" ("9521234" in the examples) starts with the same digits as
+                  the GS1
+                  Prefix in "licenseValue" of the "extendsCredential" ("952" in the examples); and
```

Comment:
This is the first use of "Extends Credential". Could we get a definition here?

At first I thought it was a property, but then there language saying "extendsCredential" was issued/revoked/etc. I'm fairly sure the property wasn't revoked, but rather we have a new type of credential in this workflow that we haven't clearly defined.

Reply:
@KDean-GS1 This still needs some cleanup.  Extends credential is not yet defined and in this section you often use `"extendsCredential"` to mean the extended credential, but that is the property that is used to reference the extended credential. It isn't the credential.

For example, you say `the issuer is the same as the subject of the "extendsCredential";` but I think what you mean is `the issuer is the same as the subject of the extended credential identified by the "extendsCredential" property. 

Also, my understanding from our conversation was that this extension mechanism is not what I initially thought it was. Perhaps explain in the list of VCs, which is an extension of which, so we have a term to refer to the extended credential, that which is identified by "extendsCredential".